mandatory/2-function-creation.js

```diff
@@ -16,7 +22,21 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (num > 100){
+    let number = 100;
+    var percentage = number + "%";
+    return percentage;
+  } else if( num % 1 !== 0){  
+    var number = parseFloat((num).toFixed(2));
+    var percentage = number + "%";
+    return percentage;
+  } else {
+    return num + "%";
+  }
+  
+  
+}
 
```

Comment:
can be  more concise, but it's nice and interesting solution to use the remainder operator